internal/pkg/addon/storage.go

```diff
@@ -259,18 +327,37 @@ func (r *RDSTemplate) MarshalBinary() ([]byte, error) {
 // NewRDSParams creates a new RDS parameters marshaler.
 func NewRDSParams() *RDSParams {
```

Comment:
nit: maybe this one too can be `RDWSParamsForRDS`, `EnvParamsForRDS`, `RDWSParamsForEnvRDS`

Reply:
done!